transform.py

```diff
@@ -1 +1,121 @@
+from datetime import datetime
+
+from pyspark.sql import SparkSession, Row
+from pyspark.sql.functions import lit
+from pyspark.sql.types import StructType, StructField, StringType, FloatType, ArrayType, DateType
+
 from kafka_interface import kafka_interface
+
+
+class Transform:
+    def __init__(self, kafka_topics, spark_master="local[*]"):
+        """
+        Initializes the Transform class with SparkSession and DataFrame schema.
+
+        Parameters:
+        - kafka_topics (list): List of Kafka topics to consume data from.
+        - spark_master (str): Spark master URL. Default is "local[*]" for local execution.
+        """
+        # Create a SparkSession
+        _spark = SparkSession.builder.master(spark_master).appName("RealTimeTransform").getOrCreate()
+
+        # Define schema for the DataFrame
+        _movie_schema = StructType([
```

Comment:
should be in parent transform class cause if it's the common entity. if it's the structure of this source it's should be in this source transformer